core/trino-spi/src/main/java/io/trino/spi/security/Identity.java

```diff
@@ -130,13 +130,15 @@ public boolean equals(Object o)
             return false;
         }
         Identity identity = (Identity) o;
```

Comment:
Can you please provide unit tests to concrete the contract against accidental changes. Possibly we could also add a javadoc.

Reply:
I added a unit test, though it's a little silly